pkg/cluster/internal/providers/docker/provision.go

```diff
@@ -122,7 +122,7 @@ func createContainer(args []string) error {
 }
 
 func clusterIsIPv6(cfg *config.Cluster) bool {
-	return cfg.Networking.IPFamily == "ipv6"
+	return cfg.Networking.IPFamily == "ipv6" || cfg.Networking.IPFamily == "DualStack"
```

Comment:
we should use the constants from config

Reply:
indeed, tech debt accumulating :shushing_face: